README.md

```diff
@@ -1,3 +1,147 @@
-# Opsgenie App
+# Mattermost/OpsGenie Integration
 
-A Opsgenie app for Mattermost.
+* [Feature summary](#feature-summary)
+* [Set up](#set-up)
+    * [Installation HTTP](#installation-http)
```

Comment:
Please update this table of contents with the updated section titles suggested below.